lightning/src/util/mod.rs

```diff
@@ -20,10 +20,10 @@ pub mod errors;
 pub mod ser;
 pub mod message_signing;
 pub mod invoice;
+pub mod chacha20;
```

Comment:
Ah, I already suspected exposing `ChaCha20` is not the best idea. I agree, iterating the 'initial' random bytes via `sha256` should be sufficient here, especially since `ChaCha20` is still used by `find_route` itself. Addressed this in https://github.com/lightningdevkit/rust-lightning/pull/1286/commits/f6793ab2192e57b44801e64900b4e68ae1fd7de1.